src/kv/deserialise.h

```diff
@@ -474,32 +370,11 @@ namespace kv
         std::move(new_maps_))
     {}
 
-    ApplyResult execute() override
+    ApplyResult apply() override
     {
-      return fn(
-        store, data, history, progress_tracker, v, changes, new_maps, hooks);
-    }
-
-    std::function<ApplyResult(
-      ExecutionWrapperStore* store,
-      const std::vector<uint8_t>& data,
-      std::shared_ptr<TxHistory> history,
-      std::shared_ptr<ccf::ProgressTracker> progress_tracker,
-      kv::Version& v,
-      OrderedChanges& changes,
-      MapCollection& new_maps,
-      kv::ConsensusHookPtrs& hooks)>
-      fn = [](
-             ExecutionWrapperStore* store,
-             const std::vector<uint8_t>& data,
-             std::shared_ptr<TxHistory> history,
-             std::shared_ptr<ccf::ProgressTracker> progress_tracker,
-             kv::Version& v,
-             OrderedChanges& changes,
-             MapCollection& new_maps,
-             kv::ConsensusHookPtrs& hooks) -> ApplyResult {
       if (!store->commit_deserialised(changes, v, new_maps, hooks))
       {
+        LOG_FAIL_FMT("receive_nonces commit_deserialized Failed");
```

Comment:
```suggestion
        LOG_FAIL_FMT("receive_nonces commit_deserialised Failed");
```

Not a comment on US vs UK spelling, just consistency with line :375.